# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../irssi/irssi-0.8.10-glibc-invalid-pointer.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://www.redhat.com/archives/fedora-extras-commits/2006-March/msg01341.html
This patch fixes a core dump which ocures when one quits irssi
Author "Dams Nade" (anvil)

--- irssi-0.8.10/src/irc/dcc/dcc.c.orig	2005-10-19 05:13:31.000000000 +0300
+++ irssi-0.8.10/src/irc/dcc/dcc.c	2006-12-18 01:32:29.021198000 +0200
@@ -58,8 +58,8 @@
 
 	pos = gslist_find_string(dcc_types, type);
 	if (pos != NULL) {
-                dcc_types = g_slist_remove(dcc_types, pos->data);
 		g_free(pos->data);
+		dcc_types = g_slist_remove(dcc_types, pos->data);
 	}
 }
 
